src/plugins/trace/dll/DataModel/QuicEvent.cs

```diff
@@ -196,7 +196,8 @@ public enum QuicEventId : ushort
         TlsErrorStatus,
         TlsMessage,
 
-        DatapathSend = 9217,
+        Temporal = 9216, // Temporary, while there are still builds out there generating this old event
+        DatapathSend,
```

Comment:
Why?

Reply:
There was `(QuicEventId)9216` hardcoded in QuicEtwEvent.cs, but this doesn't work with nameof() for QuicLTTngEvent.cs
I added additional enum to cover both cases.